src/shogun/io/SGIO.h

```diff
@@ -288,6 +288,13 @@ class SGIO
 			return location_info;
 		}
 
+		/** get last progress
```

Comment:
`@return last time progress was registered`

I.e. needs doxygen and a human understandable description (this is API doc)